csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -1563,6 +1563,36 @@ private void TestModelSequenceOfTensors()
             }
         }
 
+        [Fact]
+        private void TestModelMetadata()
+        {
+
+            string modelPath = Path.Combine(Directory.GetCurrentDirectory(), "model_with_valid_ort_config_json.onnx");
```

Comment:
I should port this test over to Java, I didn't create a model with custom fields to test out the Java metadata support.

Reply:
you can use the test model I have here.